Library/Homebrew/install.rb

```diff
@@ -40,11 +38,11 @@ def attempt_directory_creation
     end
 
     def check_cc_argv
-      return unless Homebrew.args.cc
+      return unless (cc = args.cc)
```

Comment:
Did you mean `==`?

Reply:
No, RuboCop actually makes you use parentheses for this specific case.

Reply:
Using the return value of `=` is clever but makes the code hard to reason about.

I know what the return value is in C but I don’t know it in Ruby. It’s basically impossible to look it up on https://ruby-doc.org.